less/elements.less

```
@@ -114,6 +114,7 @@ item-list-controls search {
       font-weight: normal;
       padding: .5rem 1.75rem .5rem 0;
       justify-content: start;
+      text-align: left;
```

If we wanted to use more logical properties.

```suggestion
      text-align: start;
```